tests/components/sharkiq/test_vacuum.py

```diff
@@ -0,0 +1,242 @@
+"""Test the Shark IQ vacuum entity."""
```

Comment:
youll want to make sure that vacuum.py isnt excluded from coveragerc as its not showing up in coverage stats

Reply:
Good catch.